src/ethereum/ethEngine.ts

```diff
@@ -994,7 +994,13 @@ export class EthereumEngine
     if (this.l1RollupParams != null) {
       const txData: CalcL1RollupFeeParams = {
         nonce: otherParams.nonceUsed,
-        gasPriceL1Wei: this.l1RollupParams.gasPriceL1Wei,
+        gasPriceL1Wei: ceil(
+          mul(
+            this.l1RollupParams.gasPriceL1Wei,
+            this.l1RollupParams.maxGasPriceL1Multiplier
+          ),
+          0
+        ),
```

Comment:
This math looks wrong. `ceil(x, 0)` rounds to the nearest 0? That doesn't make sense. Do you mean '1'?

Reply:
According to the docs that's how biggystring "ceil"s https://github.com/EdgeApp/biggystring/blob/798ccf7f0b12732c739a53a55d89837511e01757/README.md#rounding-functions-base-10-only

Reply:
Holy cow, that's weird! Well, whatever. Looks good to me.